gum/backend-darwin/gumobjcapiresolver.c

```diff
@@ -176,8 +176,16 @@ gum_objc_api_resolver_enumerate_matches (GumApiResolver * resolver,
   gboolean carry_on;
   GHashTable * visited_classes;
   GumObjcClassMetadata * klass;
-
-  g_regex_match (self->query_pattern, query, 0, &query_info);
+  
+  gchar * query_copy = (gchar *) query;
+  int query_len = strlen(query_copy);
+  gboolean ignore_case = (query[query_len-1] == 'i' 
+                      && query[query_len-2] == '/');
+  if (ignore_case) {
+    query_copy = g_ascii_strdown (query, strlen (query));
+    query_copy[query_len-2] = '\0';
+  }
```

Comment:
Instead of special-casing here, it would be good to modify our regex that's parsing the query. It could be as simple as extracting everything after slash, and initially just do a string compare on the result below.